### ISSUE TITLE
Diskussion der Ergebnisse

### ISSUE THREAD

**Issue by Myrt7** [whatisarthurdoing/WissenschaftlichesArbeiten#2]
Hier können wir die Ergebnisse diskutieren

**Comment by valerieweismann** [comment]
Ich finde auffällig war, dass das Interesse an Mathe die Werte 1 und 2 gar nicht angenommen hat, aber von den anderen Werten dann 3 am häufigsten aufgetreten ist.
War es festgesetzt, dass Mathestudenten immer ein Interesse von 7 hatten und Informatikstudenten ein Interesse von 3?

**Comment by DaLxx** [comment]
Ja, das ist schon mal richtig erkannt.

**Comment by Myrt7** [comment]
Zur Variable Alter: Der Mittelwert liegt bei 25.2, der Median bei 25, also würde ich vermuten diese wurde um 25.2 mit einer Varianz von 3.88 erzeugt?
Ist das Interesse am Programmieren nach unten auf 3 limitiert?
Außerdem haben alle Informatikstudenten Interesse an Mathematik 3 und Interesse an Programmieren 7.
Werden Interesse an Programmieren und Mathematik verglichen, scheint auf den ersten Blick eine negative Korrelation vorzuliegen, niedriges Interesse an Mathematik scheint mit hohem Interesse an Programmieren zusammenzuhängen. Dies fällt allerdings weg, wenn man die Informatikstudenten rausrechnet?

**Comment by DaLxx** [comment]
Der Mittelwert vom Alter war bei 25 festgelegt, also fast erkannt , die Varianz lieft da schon weiter weg.

**Comment by valerieweismann** [comment]
Ich fand noch sehr merkwürdig, dass beim Interesse an Mathe die geraden Zahlen deutlich weniger waren, als die ungeraden. Steckt da noch irgendein System dahinter, dass ich noch nicht erkannt habe?

**Comment by DaLxx** [comment]
Ja, beim Intresse am Programmieren wurde 1 und 2 nicht verwendet

**Comment by Myrt7** [comment]
Gabs noch Auffälligkeiten, die uns nicht aufgefallen sind?

**Comment by DaLxx** [comment]
Das ungerade Zahlen häufiger vor kommen, liegt wohl daran, dass die 3 und die 7 für Mathematiker und Informatiker vorgegeben waren und die Variante für die Data Sc. trägt auch dazu bei, beabsichtig war das jedoch nicht.

**Comment by DaLxx** [comment]
Habt ihr noch z.B. Wahrscheinlichkeiten für die Mathe-Lks?

**Comment by Myrt7** [comment]
Mathematik, Informatik und Statistik haben einen hohen Anteil an Mathe LK (etwa 75% ja).
Data Science hat "nur" 63%.

**Comment by DaLxx** [comment]
Eigentlich ist es nicht nach Studiengängen aufgeteilt. Sondern 70 % ja, 30% nein.
